package.json

```diff
@@ -6,6 +6,7 @@
 	"type": "module",
 	"scripts": {
 		"dev": "svelte-kit dev",
+		"debug": "node --inspect .\\node_modules\\@sveltejs\\kit\\svelte-kit.js dev",
```

Comment:
not sure about the backslashes here. I'm guessing you're a Windows user? I wonder what this will do for non-Windows users though...

Reply:
I think it’s better to use single slash `/` as it will work on Windows as well.

Reply:
Thanks for pointing that out.  I went ahead and removed this from the config because the node_modules pathing may change and this is probably a hacky way to do it.